module/metrics/example/verification/main.go

```diff
@@ -50,7 +50,7 @@ func happyPathExample() {
 
 		// starts happy path
 		t := &testing.T{}
-		verificationCollector := metrics.NewVerificationCollector(tracer, prometheus.DefaultRegisterer, logger)
+		verificationCollector := metrics.NewVerificationCollector(tracer, prometheus.DefaultRegisterer)
```

Comment:
A bit off topic from this PR, but I'm not convinced it's useful to keep this metrics example maintained any more. We can always just run `localnet` and check the metrics there.

Reply:
There are some cases that we can not easily verify on local net, e.g., currently `localnet` is default empty blocks. Without turning on the loader, the VN is quite fast enough on processing empty blocks that it is quite hard to check the mempool metrics, as normally nothing would reside that long on mempools to observe via metrics. On the other hand, turning on the loader (based on past experience) would not be very convenient as it makes the machine super slow.